doc-content/drools-docs/src/main/asciidoc/AuthoringAssets/test-scenarios-EXPECT-proc.adoc

```diff
@@ -4,10 +4,10 @@
 The *EXPECT* section defines the expected results based on the *GIVEN* input facts. That is, *GIVEN* the input facts, *EXPECT* other specified facts to be valid or entire rules to be activated. For example, with the given facts of an applicant under the age of 21 in the scenario, the *EXPECT* results could be `LoanApplication` with `approved` set to `false` (as a result of the underage applicant), or could be the activation of the `Underage` rule as a whole.
 
 .Prerequisite
-All data objects required for your test scenario have been created or imported and are listed in the *Data Objects* tab of the test scenarios designer.
+All data objects required for your test scenario have been created or imported and are listed in the *Data Objects* tab of the legacy test scenarios designer.
```

Comment:
Minor suggestion - to make the reference consistent. Maybe check if there is a label in the UI?

- legacy test scenarios designer
- *Test Scenario (Legacy)* designer

Reply:
Changed to UI label as suggested.